### PR TITLE
Set up cucumber.

### LINKED ISSUE #69: Cucumber tests

### PR DESCRIPTION
**Pull request by shaopengw**
The cucumber works now, except a strange error of pool liquidity query `await poolContract.methods.getLiquidity().call();`

> Error: Returned values aren't valid, did it run Out of Gas? You might also see this error if you are not using the correct ABI for the contract you are retrieving data from, requesting data from a block number that does not exist, or querying a node which is not fully synced.

Closes #69

### CODE REVIEW

File: test/cucumber/step-definitions/margin.steps.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  return accounts[name];
+}
+
+const transfer = async (name: string, amount: BN) => {
+  const accounts = await web3.eth.getAccounts();
+  await web3.eth.sendTransaction({ to: accountOf(name).address, from: accounts[0], value: amount});
+};
+
+const parseAmount = (amount: string): BN => {
+  const parsed = amount.replace(' ', '').replace('$', '').replace('_', '');
```
Comment (xlc) [inline test/cucumber/step-definitions/margin.steps.ts:37]: For Laminar Chain implementation, `$` means the unit is dollar, i.e. 10 ** 18. so `$10` is 10**19.

**Comment by xlc** [comment]
Maybe the ABI file needs updating? Try truffle build first?

File: test/cucumber/step-definitions/margin.steps.ts (hunk 1 above)
Comment (shaopengw) [reply test/cucumber/step-definitions/margin.steps.ts:37]: Yep a dollar should be 10**18, it doesn't work on my local env somehow, though the setup allocates 100M ether to test accounts, but somehow everyone of them only has 100.

**Comment by shaopengw** [comment]
> Maybe the ABI file needs updating? Try truffle build first?

Yep I tried build and migrate, still got the error...

**Comment by shaopengw** [comment]
Contracts are able to be called now in the step definitions.

It seems to me that the cucumber tests cannot benefit from existing truffle tests and helpers, as it can only connect to the deployed development contracts via `web3`. But since I'm not very familiar with steps of money market  minting, liquidity deposit etc, to make a liquidity pool deposit case work, I've changed features to one minimal transfer scenario just to ensure the setup work.

**Comment by gorgos** [comment]
Cool, I will take a look tomorrow. Thanks!

**Pull request merged**